src/table/utils/handle-key-press.js

```diff
@@ -178,7 +189,7 @@ export const bodyHandleKeyPress = ({
 
 export const handleLastTab = (evt, isSelectionMode, keyboard) => {
   if (isSelectionMode && evt.key === 'Tab' && !evt.shiftKey) {
-    // tab key: focus on the confirm button in the selection toolbar
+    // tab key: focus on the selection toolbar
```

Comment:
The last argument for `focusSelectionToolbar` determines if the last or first option in the toolbar will be focused. This way the focus order almost makes sense. We maybe can do something similar for the sense case, but for now we always focus the confirm button

Reply:
Do you consider the mashup? I had a test and it does not work as expected.

Reply:
In the client, when there are no pagination controls, either pressing tab or shift+table, focus on the confirm selection button. It is also not what we expected, right?

Reply:
Anyway, it fixes the issue you described. We take it step by step.

Reply:
in the client, that is what is expected. At least that is what we settled for when we did view data

Reply:
I think I tried mashup, with what keyboard setting?

Reply:
```
context: {
	keyboardNavigation: true,
	},
}
```

https://user-images.githubusercontent.com/4471489/176118397-b2551381-fb23-483d-9c31-3c03eb437f01.mov

Reply:
it is because you are tabbing from the totals, this was not merged when i was testing this I think. I'll have a look